content/ADR-259-skybox-time.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: adr
+adr: 259
+title: Skybox Time
+date: 2025-03-13
+status: Draft
+type: RFC
+spdx-license: CC0-1.0
+authors:
+  - nearnshaw
+---
+
+## Abstract
+
+The current protocol supports allowing a scene to have a fixed time of day, but it doesn't allow the creators to change it on the fly.
```

Comment:
Should we mention that's only available inside the `"worldConfiguration"` structure in the scene.json? to make clear it's only available for worlds today.

Reply:
Is that respected in E@?

Reply:
This is not currently working on E@, but we plan on implementing this together with the skyboxTime component. It is however already part of the protocol

Would be ideal to accept this property in the scene.json also if not nested inside `worldConfiguration`, so we can have it working straight up in Genesis City scenes too